### PR TITLE
fix(user): Fix the access detail page with tabView

### PR DESCRIPTION
**Pull request by mmarchal**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -68,11 +68,11 @@ class MyApp extends StatelessWidget {
...
         '/access': (context) => const AccessPage(),
         '/access/detail': (context) {
-          final args = ModalRoute.of(context)!.settings.arguments
-              as Map<String, dynamic>;
+          final args = ModalRoute.of(context)?.settings.arguments
+              as Map<String, dynamic>?;
           return AccessDetailPage(
-            id: args['id'] as String,
-            nameOfAccessUser: args['nameOfAccessUser'] as String,
+            id: args?['id'] as String? ?? 'UUS9yY2ig6e9YNcK3XQbcWPnbta2',
```
Comment (PiotrFLEURY) [inline lib/main.dart:74]: hardcoded ID ?

File: lib/main.dart [hunk 2]
```diff
@@ -68,11 +68,11 @@ class MyApp extends StatelessWidget {
...
+            nameOfAccessUser: args?['nameOfAccessUser'] as String? ?? 'Maxime',
```
Comment (PiotrFLEURY) [inline lib/main.dart:75]: hardcoded name ?

**Review by PiotrFLEURY — COMMENTED** [review]
just check hard coded values

File: lib/main.dart (hunk 2 above)
Comment (mmarchal) [reply lib/main.dart]: Forget on my part, sorry. Corrected !

File: lib/main.dart (hunk 1 above)
Comment (mmarchal) [reply lib/main.dart]: Forget on my part, sorry. Corrected !

**Review by PiotrFLEURY — APPROVED** [review]
LGTM

**Pull request merged**